cmeutils/plotting.py

```diff
@@ -0,0 +1,33 @@
+import numpy as np
+
+
+def get_histogram(data, normalize=False, bins="auto"):
+    """Bins a 1-D array of data into a histogram using
+    the numpy.histogram method.
+
+    Parameters
+    ----------
+    data : 1-D numpy.array, required
+        Array of data used to generate the histogram
+    normalize : boolean, default=False
+        If set to true, normalizes the histogram bin heights
+    bins : float, int, or str, default="auto"
+        Method used by numpy to determine bin borders.
+        Check the numpy.histogram docs for more details.
+
+    Returns
+    -------
+    bin_cetners : 1-D numpy.array
+        Array of the bin center values
+    bin_heights : 1-D numpy.array
+        Array of the bin height values
+
+    """
+    bin_heights, bin_borders = np.histogram(data, bins=bins)
+    if normalize is True:
+        bin_heights = np.array(
+                [float(i)/sum(bin_heights) for i in bin_heights]
+        )
```

Comment:
Should this be
```suggestion
        bin_heights /= max(bin_heights)
```
?
I think we want the highest peak to be 1 and all peaks to be in terms of the highest.

Reply:
I guess it depends on the goal of normalization. Maybe there should be more options via parameters?

For normalizing a probability distribution, I'd expect to normalize by the sum of the data rather than the max, so that the sum of all probabilities add up to 1. But I agree there are use cases where you want to normalize by the max.

https://stackoverflow.com/questions/26785354/normalizing-a-list-of-numbers-in-python

Reply:
I guess another option is to not normalize it, and let the user handle that after the fact however they want.

Reply:
Good idea, just added that.